### DISCUSSION TITLE
USB Audi Q3 (F3)

### DISCUSSION THREAD

**Discussion by Fahrenheit713** [Mr-MIBonk/M.I.B._More-Incredible-Bash#37] [Q&A] [answered]
Hello!
First of all thank you very much for the work you are doing!
I am actually trying to figure out if this M.I.B will be able to activate Carplay on my car.
I have an audi Q3 (F3) but I don't have Audi Smarphone Interface, that means I only have one USB  : "The Audi Q3 (type F3) is fitted with a USB-A charging socket (UE4) with up to 2.1 A as standard. This is also known as the USB hub R293 and is connected to J794".
Do you know if I have to change the USB Hub in order to get Carplay or this patch will "activate" my USB to get date from the iPhone?
Also I can't find the "hidden menus" key combinations for my MIB2+ High as it's only one touchscreen.

Thank you very much for your help!

**Comment by harman-f** [comment, marked as answer]
M.I.B only supports MHI2(Q) train.
MH2P is not supported.

**Reply by Fahrenheit713** [reply]
OK, thank you for your answer!
